watertap/property_models/activated_sludge/modified_asm2d_reactions.py

```diff
@@ -191,6 +213,12 @@ def build(self):
             domain=pyo.NonNegativeReals,
             doc="P content of inert particulate COD X_I, [kg P/kg COD]",
         )
+        self.i_PSI = pyo.Var(
+            initialize=0.00649,
+            units=pyo.units.dimensionless,
+            domain=pyo.NonNegativeReals,
+            doc="P content of inert soluble COD S_I, [kg P/kg COD]",
+        )
```

Comment:
It seems the `i_PSI` will not be used in the properties model but will be used in the translator block, so why not directly create this variable in the translator unit?

Reply:
Good point. I moved this from the property model to the translator model.